subsys/settings/src/settings_nvs.c

```diff
@@ -72,6 +73,51 @@ int settings_nvs_dst(struct settings_nvs *cf)
 	return 0;
 }
 
+#if CONFIG_SETTINGS_NVS_CACHE
+static void settings_nvs_cache_add(struct settings_nvs *cf, const char *name,
+				   uint16_t name_id)
+{
+	uint16_t name_cache = crc16_ccitt(0xffff, name, strlen(name));
```

Comment:
```suggestion
	uint16_t name_hash = crc16_ccitt(0xffff, name, strlen(name));
```
name_hash would be a better name here and elsewhere.